command-expansion-server/data/preface.txt

```diff
@@ -0,0 +1,66 @@
+/** START Preface */
```

Comment:
Lets move this file into src and rename it to CommandEDSLPreface.ts so that we can import the types from it for type checking the expansion execution code down the line - we need to adjust the command codegen to pick up that new file instead of the txt one

Reply:
It was added to packages/lib/CommandEDSLPreface.ts so I can use that path